Synergy/Frontend/src/app/app.component.ts

```diff
@@ -1,12 +1,23 @@
-import { Component } from '@angular/core';
+import {Component, OnDestroy} from '@angular/core';
+import {FirebaseAuthService} from "./services/auth/firebase-auth.service";
+import {GetUserService} from "./services/api-requests/users/get-user.service";
+import {Subscription} from "rxjs";
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrl: './app.component.css'
 })
-export class AppComponent {
-  constructor() {
+export class AppComponent implements OnDestroy {
+  userSubscription: Subscription = null!;
+  constructor(private firebaseAuthService: FirebaseAuthService, private getUserService: GetUserService) {
+    this.userSubscription = this.firebaseAuthService.currentUser$.subscribe(user => {
+      if(user)
```

Comment:
You have this subscription but i don't see anything doing anything with it or am I missing something?

Reply:
I did this so that it updates the user ngxs state when the page loads